packages/api-server/api_server/routes/tasks/scheduled_tasks.py

```diff
@@ -163,8 +163,15 @@ async def del_scheduled_tasks_event(
     if task is None:
         raise HTTPException(404)
 
-    tz = ZoneInfo(app_config.timezone) if app_config.timezone else None
-    event_date_str = event_date.replace(tzinfo=tz).isoformat()
+    utctz = ZoneInfo("UTC")
+    localtz = ZoneInfo(app_config.timezone) if app_config.timezone else None
+
+    event_date_str = event_date.replace(tzinfo=utctz).isoformat()
```

Comment:
These conversion are assuming the input is in utc, which may not be the case. The better way to fix this is to either:
* include the tz in the `except_dates`
* check the tz and convert only if it is different from local

Reply:
Good point, this would be a good step towards removing any timezone assumptions

I have just made the changes accordingly over 3 commits, https://github.com/open-rmf/rmf-web/pull/896/commits/8088ed4e9d7aa7a19513a84bec1b31786f226d61, https://github.com/open-rmf/rmf-web/pull/896/commits/1a13df309178ae058edaadb0771483a1238685bf, https://github.com/open-rmf/rmf-web/pull/896/commits/6c7af0c6c49fb0fb6a8b69b2c8e452d9ac38a34f